without it /usr/include/bits/alltypes.h defines key_t as int

--- a/tools/cert_create/Makefile
+++ b/tools/cert_create/Makefile
@@ -49,7 +49,7 @@ endif
 # from setting the OPENSSL_DIR path.
 $(eval $(call SELECT_OPENSSL_API_VERSION))
 
-HOSTCCFLAGS := -Wall -std=c99
+HOSTCCFLAGS := -Wall -std=c99 -D__DEFINED_key_t=1
 
 ifeq (${DEBUG},1)
   HOSTCCFLAGS += -g -O0 -DDEBUG -DLOG_LEVEL=40
